plugins/self-service/src/components/AAPLogoutButton/AAPLogoutButton.tsx

```
@@ -1,33 +1,51 @@
+import Typography from '@mui/material/Typography';
+import LogoutIcon from '@mui/icons-material/Logout';
 
-export const AAPLogoutButton = () => {
+class SafeBoundary extends Component<
+  PropsWithChildren<{}>,
+  { hasError: boolean }
+> {
+  constructor(props: PropsWithChildren<{}>) {
+    super(props);
+    this.state = { hasError: false };
+  }
```

ESLint

[error] constructor should be placed after getDerivedStateFromError

(react/sort-comp)